.github/workflows/stale_issues.yml

```diff
@@ -16,6 +16,8 @@ jobs:
         # that category
         ancient-issue-message: Greetings! We’re closing this issue because it has been open a long time and hasn’t been updated in a while and may not be getting the attention it deserves. We encourage you to check if this is still an issue in the latest release and if you find that this is still a problem, please feel free to comment or open a new issue.
         stale-issue-message: This issue has not received a response in 1 week. If you still think there is a problem, please leave a comment to avoid the issue from automatically closing.
+        stale-pr-message: This PR has not received a response in 1 week. If you still think there is a problem, please leave a comment to avoid the PR from automatically closing.
+        # These labels are required
```

Comment:
This line is duplicated

Reply:
One of them mentions issues and the other one mentions the PR, let me know if you I need to change the wording for the PR message.

Reply:
I meant the comment...